lib/importers/google/sessions/call_record.rb

```diff
@@ -0,0 +1,37 @@
+module Importers
+  module Google
+    module Sessions
+      class CallRecord
+        def initialize(cells)
+          @cells = cells
+        end
+
+        def params
+          {
+            delivery_partner: Partners::WEB_VISITS,
+            reporting_month: reporting_month,
+            transactions: transactions,
+            bookings: 0,
+            completions: 0
+          }
+        end
+
+        def unique_identifier
+          params.slice(:reporting_month, :delivery_partner)
+        end
+
+        def reporting_month
+          if @cells[0].to_i < 10
+            "#{@cells[1]}-0#{@cells[0]}"
+          else
+            "#{@cells[1]}-#{@cells[0]}"
+          end
+        end
```

Comment:
Also I just noticed this file doesn't have tests. Possibly the feature is enough to cover this? If it had tests describing the cell layout then possibly the above two comments become moot.